tests/integration/test_fim/test_files/test_audit_buffer/data/test_cases/cases_audit_buffer_values.yaml

```diff
@@ -0,0 +1,56 @@
+- name: queue_size_100_value_inside_range_starts_properly
+  description: Verify if value set is inside valid range it is configured and whodata starts correctly.
+  configuration_parameters:
+    QUEUE_SIZE: 100
+  metadata:
+    queue_size: 100
+    audit_starts: true
+    valid_range: true
+
+- name: queue_size_10_Lower_limit_starts_properly
+  description: Verify if value set is the lower allowed limit, whodata starts correctly
+  configuration_parameters:
+    QUEUE_SIZE: 10
+  metadata:
+    queue_size: 10
+    valid_range: true
+    audit_starts: true
+
+- name: queue_size_1048576_upper_limit_starts_properly
+  description: Verify if value set to the upper limit whodata starts correctly.
+  configuration_parameters:
+    QUEUE_SIZE: 1048576
+  metadata:
+    queue_size: 1048576
+    audit_starts: true
+    valid_range: true
+
+- name: queue_size_invalid_value_below_minimum
```

Comment:
What about negative values?

Reply:
No negative values have been tested because 10 is the lower limit of events. A lower amount causes whodata to not start because it fails the healthcheck.